modules/ROOT/pages/indexes-for-full-text-search.adoc

```diff
@@ -56,6 +56,7 @@ The commands and procedures for full-text indexes are listed in the table below:
 | Create full-text node index.
 | `+CREATE FULLTEXT INDEX ...+`
 | Create a node full-text index for the given labels and properties.
+See the xref:indexes-for-search-performance.adoc#indexes-create-indexes[`CREATE INDEX`] command for more details.
```

Comment:
Not sure how much that page has on fulltext but it covers it in general at least